### PR TITLE
[Tournament] Age of Empires 2

### PR DESCRIPTION
**Pull request by aarongerig**
This PR introduces the ruleset for the Age of Empires 2 tournament at SwitzerLAN 2023.

Question @runebaas: Is there a way to add (non-external) images?

### CODE REVIEW

**Comment by aarongerig** [comment]
@runebaas This PR is ready for review now. As long as the times and schedule stay the same, it is also ready to be merged. We'll discuss the schedule of the tournament on Discord to finalize it.

Thanks a lot for reviewing!

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,6 @@ website/public/
 website/resources/
 website/content/
 website/.hugo_build.lock
+
+# Jetbrains IntelliJ
+/.idea/
```
Comment (runebaas) [inline .gitignore:7]: Please use a global gitignore https://stackoverflow.com/questions/7335420/global-git-ignore

File: rules/tournaments/aoe2de.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+The official tournament page will be shared via the main communication channel established in the main rulebook.
+
+### Tournament Format
+
+The tournament is played in the following stages:
+
+| Stage   | Type               | Format    | Description                                                            |
+|---------|--------------------|-----------|------------------------------------------------------------------------|
+| Stage 1 | Swiss Rounds       | Bo1       | This stage will be seeded according to current 1v1 RM ELO rankings.    |
```
Comment (runebaas) [inline rules/tournaments/aoe2de.md:26]: Swiss rounds don't need seeding

File: rules/tournaments/aoe2de.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+| Stage 2 | Single Elimination | Bo1 / Bo3 | The bracket will be seeded according to the result of the first stage. |
+
+### Qualification Swiss System
+
+4 rounds are played according to the Swiss-System. The games will be reallocated after each round according to the
```
Comment (runebaas) [inline rules/tournaments/aoe2de.md:31]: I would suggest to rewrite it in the follow way:

> In the Swiss-System tournament, there will be a total of four rounds played. After each round, game pairings will be adjusted based on the current player rankings. These rankings are determined by considering the number of games won and lost by each player. Once the four rounds are completed, the top eight players in the rankings will advance to compete for the tournament victory. In the event of a tie for the eighth place after the four rounds, the ranking will be automatically calculated to determine the final eight participants.

Or potentially drop the entire section and link to the [Wikipedia page](https://en.wikipedia.org/wiki/Swiss-system_tournament).

File: rules/tournaments/aoe2de.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+current ranking. For the ranking the won and lost games count. The best eight players will then play for the tournament
+victory. If there is a tie for 8th place after the 4 rounds, the ranking will be calculated automatically.
+
+## Schedule
+
+The first appointment for the team representative will be the briefing on Saturday at 15:30.
+The tournament itself will start one hour later at 16:30.
+All players must be at the location at the start of the tournament.
+
+No accommodations will be made for other tournaments that may happen during the time of this tournament.
```
Comment (runebaas) [inline rules/tournaments/aoe2de.md:41]: This line is only valid for T1 tournaments, Other tiers should accommodate those.

File: rules/tournaments/aoe2de.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+### Game Settings
+
+The following settings must be set and checked before each game.
+
+![Game Settings for SwitzerLAN 23](https://cdn.discordapp.com/attachments/592823339643437056/1158747858854482022/game_settings.png "Game Settings")
+
+### Mods
+
+All mods are allowed.
```
Comment (runebaas) [inline rules/tournaments/aoe2de.md:67]: Wouldn't this lead to cheating?

File: rules/tournaments/aoe2de.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+## Game Account
+
+Each player plays with his own game account.
```
Comment (runebaas) [inline rules/tournaments/aoe2de.md:71]:
```suggestion
Each player plays with their own game account.
```

File: rules/tournaments/aoe2de.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+* African Clearing
+* Gold Rush
+* Four Lakes
+* Acropolis
+
+### Map Selection Process
+
+#### Swiss Rounds
+
+In the first round, the lower ranked (by 1v1 RM ELO) player has the choice of which map is played.
```
Comment (runebaas) [inline rules/tournaments/aoe2de.md:86]: Swiss rounds don't need seeding, so this should probably rewritten as

```suggestion
In the first round, the last named player can pick the map. 
```

Or alternatively, to make it fair, map banning or a roll of the dice 🎲

File: rules/tournaments/aoe2de.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+In a best-of-3 match, the first game is always played on Arabia.
+Afterward the loser of the previously played game chooses the map.
+
+## Civilization Draft
+
+Players may pick any civilization, but may not repeat any civilization in a match. This means that if, for example,
+a match lasts for three games, players will have played three different civilizations each (also known as "hidden pick,
+no repeat").
+
+## Conditions of Participation
```
Comment (runebaas) [inline rules/tournaments/aoe2de.md:101]: This section is already covered in the main rulebook, no need to repeat...

File: rules/tournaments/aoe2de.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+Each tournament participant is responsible...
+
+...that he/she attends the briefing on time.  
+...that he/she is ready for the tournament games.  
+...that he/she is registered for the tournament on the [SwitzerLAN website](https://switzerlan.ch/turnier/?id=287) as well as on Battlefy.  
+...that he/she joins the [SwitzerLAN Discord](https://discord.com/invite/AwJb9mF) server and responding to messages on there.
+
+## Playing Situations
+
+### Pausing the Game
```
Comment (runebaas) [inline rules/tournaments/aoe2de.md:112]: This is already covered in the main rulebook, no need to repeat...

File: rules/tournaments/aoe2de.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+
+When pausing the game, the reason and duration of the pause should always be briefly discussed among each other.
+
+### Restore the Game
```
Comment (runebaas) [inline rules/tournaments/aoe2de.md:116]:
```suggestion
### Disconnect or game crash
```

Then you can drop everything before the first comma in the paragraph bellow

File: rules/tournaments/aoe2de.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+
+In case of a disconnect or game crash, the player remaining in the game should save the game. The game should then be
+restored to the last available point in time.
+
+### End of Game
+
+The winner is the player who has the most points after 700 years (60 in-game minutes). Of course, a player also wins if
+the opponent surrenders prior to the time limit.
+
+### Winning the Game / Results
```
Comment (runebaas) [inline rules/tournaments/aoe2de.md:126]: This is already covered in the main rulebook, no need to repeat...

File: tasks.ps1 [hunk 12]
```diff
@@ -17,12 +17,12 @@ $contentFolder = "$websiteFolder/content"
...
 $nl = [System.Environment]::NewLine
 
-$finalPages = ('Main Rulebook','osu!','Rocket League','Hearthstone','Overwatch 2','League of Legends','Apex Legends','Valorant')
+$finalPages = ('Main Rulebook','osu!','Rocket League','Hearthstone','Overwatch 2','League of Legends','Apex Legends','Valorant','Age of Empires 2')
 
 ####
 # Shared Functions
 ####
-function CreateContentSkelleton() {
+function CreateContentSkeleton() {
```
Comment (runebaas) [inline tasks.ps1:25]: Nice catch!

**Review by runebaas — CHANGES_REQUESTED** [review]

File: rules/tournaments/aoe2de.md (hunk 5 above)
Comment (aarongerig) [reply rules/tournaments/aoe2de.md:67]: So far in AoE2 there has not been a mod that has unfairly affected the game. I therefore assume that there is currently no mod that could be considered cheating. However, I have clarified the text a bit to be prepared for all cases.

**Comment by aarongerig** [comment]
@runebaas Done!

**Review by runebaas — APPROVED** [review]

**Comment by aarongerig** [comment]
@runebaas All good now.

**Pull request merged**